patches/TerrariaNetCore/Terraria/Terraria.csproj.patch

```diff
@@ -66,7 +79,11 @@
 -			<HintPath Condition="$(DefineConstants.Contains('Mono'))">Libraries/Mono/System.Windows.Forms.dll</HintPath>
 -		</Reference>
 -		<Reference Include="WindowsBase" />
++
++		<PackageReference Include="Steamworks.NET" Version="20.1.0" />
  	</ItemGroup>
++
++	<!-- Embedded Resources -->
```

Comment:
Please avoid reorganizing the project file too much. We still need to minimize diffs. It's not necessary to add comments for each `ItemGroup` or `PropertyGroup` imo

Reply:
Unlike in the ancient PR, this is done only on top level, but still makes an insane difference for navigating in the clunky format that is XML. 
![image](https://github.com/tModLoader/tModLoader/assets/6795251/17a6beaa-956f-43f3-b9e8-cf0e970659a6)

I don't agree that we should always be choosing patch minimization *in a `.csproj`* over providing the patcher and ourselves more context, making things easier to repair.

Reply:
On reflection and further review of the diffs, I believe the changes are acceptable. Although I'd prefer if the `Debug/Release/Server` assembly name diffs were more similar.

Reply:
Debug&Release conditions are inserted at `Terraria`, but `Server` comes from `decompiled`. It should take three minutes to update `Terraria.ModLoader.Setup.DecompileTask` to write a csproj closer to this one, minimizing diffs for every step, but that's better off done for 1.4.5/1.5 to not require redecompilations.